CRM/Upgrade/Incremental/php/FiveFiftyNine.php

```diff
@@ -21,13 +21,22 @@
  */
 class CRM_Upgrade_Incremental_php_FiveFiftyNine extends CRM_Upgrade_Incremental_Base {
 
+  public function setPreUpgradeMessage(&$preUpgradeMessage, $rev, $currentVer = NULL): void {
+    if ($rev === '5.59.alpha1') {
+      if (empty(CRM_Core_Config::singleton()->userFramework->is_wordpress)) {
```

Comment:
userFramework is a string, but also the original issue was about wordpress, right? I can see you're reasoning but I'm struggling with whether they should or shouldn't also be notified.

Reply:
oh dang - userSystem instead of userFramework

My understanding is we are making drupal the same as wordpress here - so no change for wordpress

Reply:
I guess I would break it into 3 groups. Considering the situation where you have bad smarty before the upgrade:
* Drupal: Messages are going out, just might have some mangling which people may or may not notice. After upgrade: "Civi broke my mailings."
* Wordpress: Messages not going out, and they know that, but there's no error messages and they don't know why and have either worked around it or haven't solved it yet. After upgrade: Messages not going out but now there's a message. This group doesn't need an upgrade notice.
* Wordpress: Messages not going out, but they haven't noticed. After upgrade: "Civi broke my mailings" is their initial reaction, and then they may or may not notice they weren't going out before either, but they'll get to that realization quicker with an upgrade notice.

So is it worth it to try to reach that 3rd group at the expense of telling the 2nd group something they don't really need to read.

Reply:
@demeritcowboy let's keep scenario 3 out of scope of this PR. I'm not quite too sure how we would address that & it probably needs more thought but it's more a status check than an upgrade notice